packages/ui/utils/fetchMaxAmount.ts

```diff
@@ -48,7 +48,16 @@ export const fetchMaxAmount = async (
   comptrollerAddress?: string
 ): Promise<BigNumber> => {
   if (mode === FundOperationMode.SUPPLY) {
```

Comment:
We also need this feature for `FundOperationMode.BORROW` for `BorrowCaps`

Reply:
I will do it in separated PR